ccsljava_xdsml/plugins/org.eclipse.gemoc.execution.concurrent.ccsljavaxdsml.ui/src/main/java/org/eclipse/gemoc/execution/concurrent/ccsljavaxdsml/ui/builder/GemocLanguageDesignerBuilder.java

```diff
@@ -44,7 +44,7 @@
 import org.eclipse.gemoc.commons.eclipse.pde.manifest.ManifestChanger;
```

Comment:
Could be good to rename this class for clarity.

Reply:
is MoccmlLanguageDesignerBuilder a better name ?  (+ rename GemocLanguageDesignerNature -> MoccmlLanguageDesignerNature too)  

I also like MoccmlLanguageProjectBuilder / MoccmlLanguageProjectNature  but this naming should be reflected on the main generic builder too (when it will move from sequential builder to generic :wink: )

Reply:
Yes I would find these names better and clearer! I think it's OK if we only rename here for now.